book/website/collaboration/inclusive-events/inclusive-events-planning.md

```diff
@@ -1,31 +1,362 @@
+
+(cl-inclusive-events-planning-safeguarding)=
+### Safeguarding
+
+Although the term "people with lived experience" encompasses an intentionally wide range of circumstances, depending on your field and your event, 
+the specific attendees with lived experience may need additional safeguarding depending on what your event's "lived experience" is focused on. 
+
+In the UK, Safeguarding refers to the protection of people's health, wellbeing and human rights, and the proactive support necessary for people to live free from harm, 
+abuse and neglect (cite: Quality Care Commission or NHS - https://www.england.nhs.uk/safeguarding/about/). 
+Depending on your country and organization, "safeguarding" may be part of your Risk Assessment processes, or may be called something else entirely. 
+
+Minors and people under the age of 18, people with lived experience of autism (autistic people), people with lived experience of dementia and other neurodegenerative disorders, cognitive impairments, a
+nd/or prior experience of abuse may be at a greater risk of being taken advantage of and therefore require additional considerations and protections if attending your event. 
+
+Your event may want to consider having a "Designated Safeguarding Officer" (often referred to as either a "DSO" or "Designated Safeguarding Lead" in the UK) as part of safeguarding procedures. 
+A Designated Safeguarding Officer is an official role who will handle and manage safeguarding reports and is a point of contact throughout the event. 
+They will be the point of contact for any legal, medical, or social services that may need to be contacted and is generally a senior organizer or member of staff.
+
+You may consider organizing safeguarding training for staff and organizers who will be attending the event which can help to prepare and train organizers in identifying the types of abuse and indicators, 
+as well as how to report concerns about safeguarding and manage disclosure at the event. Depending on your event, the disclosure of experiencing harm, as well as the medical conditions and health status of attendees, 
+may be shared as part of discussion with people with lived experience. It is important to consider how you will handle potential disclosure and what you will do to uphold the personal safety of an 
+individual who discloses personal information unknowingly in a public forum at your event. 
+
+By creating a safeguarding policy that covers the reporting and escalation pathways and making this accessible to staff and attendees you will be better prepared to manage any events of disclosure 
+and uphold the necessary safeguarding for your attendees. 
+
+Please also remember that safeguarding applies to both attendees and staff and it is important to consider the physical and psychological safety of staff at your event, as well as your attendees.
+
+(cl-inclusive-events-planning-before-event)=
+## Before the Event
```

Comment:
This confused me a bit now during the review, if only this bit is _before the event_, how does it differ from everything up to here? Would it be worth folding it in above? (tbf: this might just be a result of me reviewing directly in the raw markdown instead of a rendered version!)

Reply:
I think we can change the wording. 

I tried to separate out H2: initial planning i.e. 6months or longer before event, and the H2: before the event i.e. a few weeks before. 

I can revise the phrasing. It's a really weird semantic tangle to try say "pre-planning, planning, lead up planning, ... etc" as when does pre-planning become planning. 

I've have another think and see what I can come up with to make it less confusing.

Reply:
Perfect, now I got the distinction! Maybe just adding those timelines in brackets after those in the headings could work as an easy fix? :)

Reply:
I also thought maybe "final planning considerations" could also work as a heading. It might be one that we get more feedback on and find what is clearest.